Add guidance for dealing with Bluetooth interference

Hedda opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->

Extending bluetooth.markdown Bluetooth interference tips based on some of advice from related Zigbee PR for ZHA -> https://github.com/home-assistant/home-assistant.io/pull/18864

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Hedda committed:
Update bluetooth.markdown Bluetooth interference

Update bluetooth.markdown extending Bluetooth interference tips

Hedda commented:
hmm, Lint report 1 problem (1 error, 0 warnings) /  1 fixable problem but does not report which line or which word is the problem.

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+
+#### Understanding common root causes of problems in Bluetooth radio setups
+
+- Bluetooth adapter hardware:
+  - Poor placement of the Bluetooth adapter and/or wrong orientation of Bluetooth adapter antenna.
+  - Old, outdated Bluetooth adapter hardware or poor Bluetooth adapter antenna.
+  - Poor or outdated Bluetooth adapter firmware on the Bluetooth adapter.
+-  Bluetooth adapters and Bluetooth devices sensitive to all types of RFI/EMI/EMF interference:
+  - Electromagnetic interference (EMI) from electronics caused by Electromagnetic Fields (EMF).
+  - USB 3.0 ports and computer peripherals are known culprits of EMI/EMF disrupting. Ref. [1](https://www.usb.org/sites/default/files/327216.pdf) and [1](https://www.unit3compliance.co.uk/2-4ghz-intra-system-or-self-platform-interference-demonstration/).
+  - 2.4 GHz Radio Frequency Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points, or others.
+  - While unsual as Bluetooth is designed to coexist with Wi-Fi, their stronger signal can interfere.
+  - To play it safe best is to try to place your Bluetooth adapter awasy from Wi-Fi access-points.
```
Some of this is redundant with the above section. Can you make it a bit more succinct?

Hedda replied:
Do you mean removing the whole bullet point list and instead extend this above section? "**_Most important is to know that other devices that also broadcast on the 2.4 GHz radio band, like USB3 devices (and their cable connections), or other devices are known to affect Bluetooth radio reception. Especially external SSD drives with USB3 cables are known to block the Bluetooth signal. Also, metal casings can decrease the Bluetooth performance of internal Bluetooth Adapters. Recommendation is therefore to at least place USB3 devices and their cables (SSD, etc.) as far away as possible from your Bluetooth adapter, e.g., by using your Bluetooth adapter with a long extension cable that has proper shielding._**"

bdraco replied:
I'm mostly concerned about the duplication between

#### Simple actions that should improve most Bluetooth setups
and 
#### Understanding common root causes of problems in Bluetooth radio setups


The first section has `USB 3.0 ports/peripherals are infamously known to cause EMI/EMF interference.`

the second section has `USB 3.0 ports and computer peripherals are known culprits of EMI/EMF disrupting. Ref. [1](https://www.usb.org/sites/default/files/327216.pdf) and [1](https://www.unit3compliance.co.uk/2-4ghz-intra-system-or-self-platform-interference-demonstration/).`

Hedda replied:
So maybe merge into a single large bullet point list and as such removing the whole second section is better? Like this:

#### Simple actions that should improve most Bluetooth setups and common root causes of interference to be aware of

- Bluetooth adapter hardware:
  - Bad performance from old/outdated Bluetooth adapter hardware or poor Bluetooth adapter antenna.
    - Buy and use a (good quality) Bluetooth USB adapter based on a newer/modern chip hardware.
      - If possible consider buying/upgrading to a Bluetooth adapter that has an external antenna. 
      - While older adapters might work they could have obsolete hardware and/or old firmware.
  - Poor or outdated Bluetooth adapter firmware on the Bluetooth adapter.
    - Update to a later released Bluetooth chip firmware on the Bluetooth adapter.
      - Updating firmware is usually easy if one is provide by the manufacturer or the chip maker.
- Bluetooth adapters are RFI sensitive and can be very susceptible to all types of EMI/EMF interference:
  - Poor placement of the Bluetooth adapter and/or wrong orientation of Bluetooth adapter antenna.
    - Use a long USB extension cable to place Bluetooth adapter away from interference and obstacles.
      - Make sure the USB extension cable has proper shielded (thick cables usually have this).
        - Preferably by a longer extension cable specified to have proper shielding and ferrite clamps.
        - Using a USB extension cable also make it easier to orient the Bluetooth adapter/antenna.
    - Try different physical placement and orientations of Bluetooth adapter or its antenna.
      - Optimal placement of Bluetooth adapter is as much in middle of the house as possible.
      - Try to place the Bluetooth adapter at some distance way from walls, ceilings and floors.
      - Try different orientations of the adapters external antenna (or whole Bluetooth adapter).
  - USB 3.0 ports/computers/peripherals are known culprits of RFI/EMI/EMF disruption. (See Ref. [1](https://www.usb.org/sites/default/files/327216.pdf) and [2](https://www.unit3compliance.co.uk/2-4ghz-intra-system-or-self-platform-interference-demonstration/)).
    - Make sure to only connect the Bluetooth USB adapter to a USB 2.0 port (and not to a USB 3.x port). 
      - If your computer only has a USB 3.x port then connect the adapter via a powered USB 2.0 hub.
        - A USB 2.0 hub will convert USB 3.0 to a USB 2.0 port.
        - Buy a USB 2.0 hub that has an external power supply to ensure fulfil power requirements.
    - Shield close unshielded computers and unshielded peripherals by using metal enclosures/chassis.
      - Single-board-computers and USB 3.x hard drives are especially known as source of EMI/EMF.
      - Also be sure to use properly shielded USB cables for any such peripherals/devices too.
  - 2.4 GHz RF Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points, or other type of devices.
    - While unusual as Bluetooth is designed to coexist with Wi-Fi, their stronger signal can interfere.
      - To play it safe best to try to place your Bluetooth adapter away from Wi-Fi access-points.
    - Place away from electrical/power wires/cables and power supplies and even household appliances.

bdraco replied:
Yes thats better.

Hedda replied:
OK thanks for the feedback, updated now.

Hedda committed:
Update bluetooth.markdown based on feedback

Update bluetooth.markdown based on feedback on bullet point list with actions to avoid interference and optimize setup.

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,44 @@ Deleting the config entry for this integration will release control of the adapt
-- Try to place USB3 devices (SSD, etc.) as far away as possible from your Bluetooth Adapter, e.g., by using an extension cable.
-- Use a USB3 extension cable with proper shielding and ferrite clamps.
-- Use a (good quality) external Bluetooth adapter with an antenna.
+Poor signal quality or sources of interference for radios can lead to transmission/reception loss or connection problems and show symptoms such as or errors/failures when sending and/or receiving Bluetooth messages that can cause significant degradation in performance. Below are some basic but essential tips for how to get a good setup starting point to achieve an overall better signal quality +  ...
```
```suggestion
Sources of interference for radios can lead to transmission/reception loss or connection problems and show symptoms such as errors/failures when sending and receiving Bluetooth messages that can cause significant degradation in performance. Below are some basic but essential tips for getting a good setup starting point to achieve better signal quality, coverage, and extended range.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,44 @@ Deleting the config entry for this integration will release control of the adapt
-- Use a (good quality) external Bluetooth adapter with an antenna.
+Poor signal quality or sources of interference for radios can lead to transmission/reception loss or connection problems and show symptoms such as or errors/failures when sending and/or receiving Bluetooth messages that can cause significant degradation in performance. Below are some basic but essential tips for how to get a good setup starting point to achieve an overall better signal quality +  ...
+
+Following all these optimization tips below should significantly improve reception of your Bluetooth radio adapter and can resolve or avoid many known issues caused by a interference or poor placement of your Bluetooth radio adapter or devices. The most important knowledge this tries to impart relates to the basics on working around the well known limitations of low-power 2.4 GHz digital radio.
```
```suggestion
Following all these optimization tips below should significantly improve the reception of your Bluetooth radio adapter. The below insights describe working around the well-known limitations of low-power 2.4 GHz digital radio. It can resolve or avoid many known issues caused by interference or poor placement of your Bluetooth radio adapter or devices.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,44 @@ Deleting the config entry for this integration will release control of the adapt
+
+Following all these optimization tips below should significantly improve reception of your Bluetooth radio adapter and can resolve or avoid many known issues caused by a interference or poor placement of your Bluetooth radio adapter or devices. The most important knowledge this tries to impart relates to the basics on working around the well known limitations of low-power 2.4 GHz digital radio.
+
+Most important is to understand and keep in mind that that other computers/peripherals/devices unintentially or intentionally generate [electromagnetic interference (also known as EMI/EMI/RMI)](https://en.wikipedia.org/wiki/Electromagnetic_interference) as blocking radio signals transmissions on on the 2.4 GHz radio band frequency, which in practice acts as radio jammers that block the wireless c ...
```
```suggestion
Computers, peripherals, and devices generate [electromagnetic interference (also known as EMI/EMI/RMI)](https://en.wikipedia.org/wiki/Electromagnetic_interference), which can interfere with signals transmissions on the 2.4 GHz radio band frequency, and degrade the wireless communication with your Bluetooth adapter/devices.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,44 @@ Deleting the config entry for this integration will release control of the adapt
+
+Most important is to understand and keep in mind that that other computers/peripherals/devices unintentially or intentionally generate [electromagnetic interference (also known as EMI/EMI/RMI)](https://en.wikipedia.org/wiki/Electromagnetic_interference) as blocking radio signals transmissions on on the 2.4 GHz radio band frequency, which in practice acts as radio jammers that block the wireless c ...
+
+For example, unshielded USB 3.x computers/peripherals/devices and their cables are especially infamously known to affect 2.4 GHz radio reception. The general recommendation is therefore to add all-metal shielding to computers/peripherals/devices or at the very least try to place your Bluetooth adapter far away as possible from any potential sources of EMI/EMI/RMI (e.g., by connecting your Bluetoo ...
```
```suggestion
For example, unshielded USB 3 port and their cables are especially infamously known to affect 2.4 GHz radio reception. Place your Bluetooth adapter far away as possible from any potential sources of EMI/EMI/RMI by using a long, adequately shielded USB extension cable.
```

bdraco replied:
This advice appears at least 3 times on this page.

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,44 @@ Deleting the config entry for this integration will release control of the adapt
+
+For example, unshielded USB 3.x computers/peripherals/devices and their cables are especially infamously known to affect 2.4 GHz radio reception. The general recommendation is therefore to add all-metal shielding to computers/peripherals/devices or at the very least try to place your Bluetooth adapter far away as possible from any potential sources of EMI/EMI/RMI (e.g., by connecting your Bluetoo ...
+
+#### Simple actions that should improve most Bluetooth setups and common root causes of interference to be aware of
```
```suggestion
#### Simple actions that should improve most Bluetooth setups and common root causes of interference
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,44 @@ Deleting the config entry for this integration will release control of the adapt
+
+- Bluetooth adapter hardware:
+  - Bad performance from old/outdated Bluetooth adapter hardware or poor Bluetooth adapter antenna:
+    - Buy and use a (good quality) Bluetooth USB adapter based on a newer/modern chip hardware.
```
```suggestion
    - Buy and use a (good quality) Bluetooth USB adapter based on newer/modern chip hardware.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,44 @@ Deleting the config entry for this integration will release control of the adapt
+- Bluetooth adapter hardware:
+  - Bad performance from old/outdated Bluetooth adapter hardware or poor Bluetooth adapter antenna:
+    - Buy and use a (good quality) Bluetooth USB adapter based on a newer/modern chip hardware.
+      - If possible consider buying/upgrading to a Bluetooth adapter that has an external antenna. 
```
```suggestion
      - Consider a Bluetooth adapter that has an external antenna. 
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,44 @@ Deleting the config entry for this integration will release control of the adapt
+  - Bad performance from old/outdated Bluetooth adapter hardware or poor Bluetooth adapter antenna:
+    - Buy and use a (good quality) Bluetooth USB adapter based on a newer/modern chip hardware.
+      - If possible consider buying/upgrading to a Bluetooth adapter that has an external antenna. 
+      - While older adapters might work they could have obsolete hardware and/or old firmware.
```
```suggestion
      - While older adapters might work, they could have obsolete hardware or old firmware, which prevents reliable operation.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,44 @@ Deleting the config entry for this integration will release control of the adapt
+      - If possible consider buying/upgrading to a Bluetooth adapter that has an external antenna. 
+      - While older adapters might work they could have obsolete hardware and/or old firmware.
+  - Poor or outdated Bluetooth adapter firmware on the Bluetooth adapter:
+    - Update to a later released Bluetooth chip firmware on the Bluetooth adapter.
+      - Updating firmware is usually easy if one is provide by the manufacturer or the chip maker.
```
```suggestion
    - Update to a later released Bluetooth chip firmware on the Bluetooth adapter. Updating firmware is usually straightforward if the manufacturer or the chip maker provides one.
```

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda commented:
@bdraco Thanks for the review! All suggested changes have been committed to the PR now.

Hedda committed:
Merge branch 'current' into patch-69

bdraco commented:
Looks much better. I'm on holiday this week but I'll do another review pass when I get back after 2022.9 is released.  Thanks for your patience and contribution

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+    - Make sure to only connect the Bluetooth USB adapter to a USB 2.0 port (and not to a USB 3.x port). 
+      - If your computer only has a USB 3.x port then connect the adapter via a powered USB 2.0 hub:
+        - A USB 2.0 hub will convert USB 3.0 to a USB 2.0 port and thus you avoid USB 3.0 EMF.
+          - USB 2.0 hub that use an external power supply ensure power requirements is fulfilled.
```
```suggestion
          - A USB 2.0 hub that uses an external power supply will ensure power requirements are fulfilled.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+      - If your computer only has a USB 3.x port then connect the adapter via a powered USB 2.0 hub:
+        - A USB 2.0 hub will convert USB 3.0 to a USB 2.0 port and thus you avoid USB 3.0 EMF.
+          - USB 2.0 hub that use an external power supply ensure power requirements is fulfilled.
+    - Shield to unshielded computers/peripherals/devices by adding all-metal enclosures/chassis/casings.
```
```suggestion
    - Shield any unshielded computers/peripherals/devices by adding all-metal enclosures/chassis/casings.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+      - Also be sure to use properly shielded USB cables for any such peripherals/devices too.
+  - 2.4 GHz RF Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points, or other type of devices.
+    - While unusual as Bluetooth is designed to coexist with Wi-Fi, their stronger signal can interfere.
+      - To play it safe best to try to place your Bluetooth adapter away from Wi-Fi access-points.
```
```suggestion
      - To play it safe, try to place your Bluetooth adapter away from Wi-Fi access points.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+        - Be aware metal casings can decrease performance of internal/built-in Bluetooth adapters.
+      - Also be sure to use properly shielded USB cables for any such peripherals/devices too.
+  - 2.4 GHz RF Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points, or other type of devices.
+    - While unusual as Bluetooth is designed to coexist with Wi-Fi, their stronger signal can interfere.
```
```suggestion
    - While Bluetooth is designed to coexist with Wi-Fi, its stronger signal can interfere.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+      - Single-board-computers and USB 3.x hard drives are especially known as source of EMF/EMI/RFI.
+        - Be aware metal casings can decrease performance of internal/built-in Bluetooth adapters.
+      - Also be sure to use properly shielded USB cables for any such peripherals/devices too.
+  - 2.4 GHz RF Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points, or other type of devices.
```
```suggestion
  - 2.4 GHz RF Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points or other devices:
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+          - USB 2.0 hub that use an external power supply ensure power requirements is fulfilled.
+    - Shield to unshielded computers/peripherals/devices by adding all-metal enclosures/chassis/casings.
+      - Single-board-computers and USB 3.x hard drives are especially known as source of EMF/EMI/RFI.
+        - Be aware metal casings can decrease performance of internal/built-in Bluetooth adapters.
```
```suggestion
        - Be aware metal casings can decrease the performance of internal/built-in Bluetooth adapters.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+    - Shield to unshielded computers/peripherals/devices by adding all-metal enclosures/chassis/casings.
+      - Single-board-computers and USB 3.x hard drives are especially known as source of EMF/EMI/RFI.
+        - Be aware metal casings can decrease performance of internal/built-in Bluetooth adapters.
+      - Also be sure to use properly shielded USB cables for any such peripherals/devices too.
```
```suggestion
      - Also, be sure to use adequately shielded USB cables for any such peripherals/devices too.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+  - 2.4 GHz RF Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points, or other type of devices.
+    - While unusual as Bluetooth is designed to coexist with Wi-Fi, their stronger signal can interfere.
+      - To play it safe best to try to place your Bluetooth adapter away from Wi-Fi access-points.
+    - Place away from any electrical/power wires/cables, power supplies, and household appliances.
```
```suggestion
    - Place Bluetooth adapters far away from electrical/power wires/cables, power supplies, and household appliances.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+  - USB 3.0 ports/computers/peripherals are known culprits of RFI/EMI/EMF disruption. (See Ref. [1](https://www.usb.org/sites/default/files/327216.pdf) and [2](https://www.unit3compliance.co.uk/2-4ghz-intra-system-or-self-platform-interference-demonstration/)).
+    - Make sure to only connect the Bluetooth USB adapter to a USB 2.0 port (and not to a USB 3.x port). 
+      - If your computer only has a USB 3.x port then connect the adapter via a powered USB 2.0 hub:
+        - A USB 2.0 hub will convert USB 3.0 to a USB 2.0 port and thus you avoid USB 3.0 EMF.
```
```suggestion
        - A USB 2.0 hub will convert USB 3.0 to a USB 2.0 port and avoid USB 3.0 EMF.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+    - Try different physical placement and orientations of Bluetooth adapter or its antenna:
+      - Optimal placement of Bluetooth adapter is as much in middle of the house as possible.
+      - Try to place the Bluetooth adapter at some distance way from walls, ceilings and floors.
+      - Try different orientations of the adapters external antenna (or whole Bluetooth adapter).
```
```suggestion
      - Try different orientations of the adapter's external antenna (or whole Bluetooth adapter).
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+        - Using a USB extension cable also make it easier to orient the Bluetooth adapter/antenna.
+    - Try different physical placement and orientations of Bluetooth adapter or its antenna:
+      - Optimal placement of Bluetooth adapter is as much in middle of the house as possible.
+      - Try to place the Bluetooth adapter at some distance way from walls, ceilings and floors.
```
```suggestion
      - Try to place the Bluetooth adapter at some distance away from walls, ceilings, and floors.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+        - Preferably by a longer extension cable specified to have proper shielding and ferrite clamps.
+        - Using a USB extension cable also make it easier to orient the Bluetooth adapter/antenna.
+    - Try different physical placement and orientations of Bluetooth adapter or its antenna:
+      - Optimal placement of Bluetooth adapter is as much in middle of the house as possible.
```
```suggestion
      - The optimal placement of the Bluetooth adapter is close to the middle of the house as possible.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+      - Make sure the USB extension cable has proper shielded (thick cables usually have this).
+        - Preferably by a longer extension cable specified to have proper shielding and ferrite clamps.
+        - Using a USB extension cable also make it easier to orient the Bluetooth adapter/antenna.
+    - Try different physical placement and orientations of Bluetooth adapter or its antenna:
```
```suggestion
    - Try different physical placement and orientations of the Bluetooth adapter or its antenna:
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+    - Use a long USB extension cable to place Bluetooth adapter away from interference and obstacles.
+      - Make sure the USB extension cable has proper shielded (thick cables usually have this).
+        - Preferably by a longer extension cable specified to have proper shielding and ferrite clamps.
+        - Using a USB extension cable also make it easier to orient the Bluetooth adapter/antenna.
```
```suggestion
        - A USB extension cable makes orienting the Bluetooth adapter/antenna easier.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+- Bluetooth adapters are RFI sensitive and can be very susceptible to all types of EMI/EMF interference:
+  - Poor placement of the Bluetooth adapter and/or wrong orientation of Bluetooth adapter antenna:
+    - Use a long USB extension cable to place Bluetooth adapter away from interference and obstacles.
+      - Make sure the USB extension cable has proper shielded (thick cables usually have this).
+        - Preferably by a longer extension cable specified to have proper shielding and ferrite clamps.
```
```suggestion
      - Ensure the USB extension cable is adequately shielded (thick cables usually have this).
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+    - Update to a later released Bluetooth chip firmware on the Bluetooth adapter. Updating firmware is usually straightforward if the manufacturer or the chip maker provides one.
+- Bluetooth adapters are RFI sensitive and can be very susceptible to all types of EMI/EMF interference:
+  - Poor placement of the Bluetooth adapter and/or wrong orientation of Bluetooth adapter antenna:
+    - Use a long USB extension cable to place Bluetooth adapter away from interference and obstacles.
```
```suggestion
    - Use a long USB extension cable to place the Bluetooth adapter away from interference and obstacles.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+  - Poor or outdated Bluetooth adapter firmware on the Bluetooth adapter:
+    - Update to a later released Bluetooth chip firmware on the Bluetooth adapter. Updating firmware is usually straightforward if the manufacturer or the chip maker provides one.
+- Bluetooth adapters are RFI sensitive and can be very susceptible to all types of EMI/EMF interference:
+  - Poor placement of the Bluetooth adapter and/or wrong orientation of Bluetooth adapter antenna:
```
```suggestion
  - Poor placement of the Bluetooth adapter or wrong orientation of Bluetooth adapter antenna:
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+      - Consider a Bluetooth adapter that has an external antenna. 
+      - While older adapters might work, they could have obsolete hardware or old firmware, which prevents reliable operation.
+  - Poor or outdated Bluetooth adapter firmware on the Bluetooth adapter:
+    - Update to a later released Bluetooth chip firmware on the Bluetooth adapter. Updating firmware is usually straightforward if the manufacturer or the chip maker provides one.
```
```suggestion
    - Update to the latest Bluetooth chip firmware on the Bluetooth adapter. Updating firmware is usually straightforward if the manufacturer or the chip maker provides one.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+
+- Bluetooth adapter hardware:
+  - Bad performance from old/outdated Bluetooth adapter hardware or poor Bluetooth adapter antenna:
+    - Buy and use a (good quality) Bluetooth USB adapter based on newer/modern chip hardware.
```
```suggestion
    - Buy and use a supported Bluetooth USB adapter based on newer/modern chip hardware.
```

bdraco commented on source/_integrations/bluetooth.markdown:
```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+- Bluetooth adapter hardware:
+  - Bad performance from old/outdated Bluetooth adapter hardware or poor Bluetooth adapter antenna:
+    - Buy and use a (good quality) Bluetooth USB adapter based on newer/modern chip hardware.
+      - Consider a Bluetooth adapter that has an external antenna. 
```
```suggestion
      - Consider a Bluetooth adapter that has an external antenna.
```

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda committed:
Update source/_integrations/bluetooth.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

Hedda commented:
@bdraco Thanks for the review, I have now committed all your suggested changes.
